src/app/models/__tests__/form.server.model.spec.ts

```diff
@@ -1676,5 +1676,45 @@ describe('Form Model', () => {
         expect(updatedForm).toBeNull()
       })
     })
+
+    describe('updateFormCollaborators', () => {
+      it('should return the form with an updated list of collaborators', async () => {
+        // Arrange
+        const newCollaborators = [
+          {
+            email: `fakeuser@${MOCK_ADMIN_DOMAIN}`,
+            write: false,
+          },
+        ]
+
+        // Act
+        const actual = await validForm.updateFormCollaborators(newCollaborators)
+
+        // Assert
+        const actualPermissionsWithoutId = map(
```

Comment:
hahahahahaha @ using `_.map` instead of native map

Reply:
y u laff